news/2024/2024-07-01-the-followpoint-neuro-sama-the-ai-vtuber-that-invaded-osu.md

```diff
@@ -0,0 +1,118 @@
+
+*What other rhythm games besides osu! have you played?*
+
+> I personally haven't tried too many other rhythm games, although I've been known to enjoy some Beat Saber from time to time. I've honestly not really played many games in general recently, with the exception of games on stream.
+
+*How did you come up with OG Neuro (AI learning to play osu!) idea and why exactly in osu! of all games? Also I would like to know what made you decide to revive Neuro as a VTuber in 2022?*
+
+> I was just getting started with learning AI in 2019, and I happened to be playing a lot of osu! at the time. I'd always found it super interesting and cool when people made AI's to play games, and I had an idea for how to do it with osu!. The original AI took a few days to make and I thought it would be a cool idea to stream it - to track its progress. Eventually after improving the AI, its performance stagnated and I wasn't sure what to do with it since I had no idea what I was doing at the time. Two years later, I had the idea to make an AI VTuber although it wasn't until early 2022 that I had the idea to combine the original Neuro-sama with this concept.
+
+*How did you feel about Neuro's sudden rise in popularity and how do you think most of osu! community reacted to her?*
+
+> I was very unprepared for Neuro's sudden rise in popularity and I certainly did not think she would make it as far as she has done today. I think there's a lot of overlap in the osu! community with tech and anime, so I feel she was well received, and it was very surreal seeing a community I was somewhat involved in become interested and supportive of something I had created.
+>
+> Whenever I'd reach a new viewer peak I'd think "surely that's it", and I just kept getting surprised over and over. I started streaming on a Monday, and by Friday, she was averaging over 2000 viewers. My previous streams in 2019 had peaked at a few hundred viewers, and I had hoped to replicate this. I remember on one of the first few streams with Neuro-sama, BTMC opened the stream live to a couple thousand viewers and my heart was racing because of it. I genuinely never expected anything like this, but I'm so thankful for it.
+
+*What would you call the hardest part in making Neuro what she is now, and what have you learned during the process?*
+
+> I think the hardest part of making Neuro was giving her personality. This isn't something I talk about often but is arguably the biggest thing that separates Neuro from a lot of the other AIs out there. AIs like ChatGPT are trained to act as assistants and tools for humans, but Neuro is trained differently. This allows her to feel more independent, and that whole process definitely taught me a lot.
+
+*And what part of Neuro did you enjoy working on the most?*
```

Comment:
```suggestion
*What part of Neuro did you enjoy working on the most?*
```

Reply:
I disagree here, it's like a second part of previous question

Reply:
sentences don't start with an `and`, if you want to make it a follow-up questions make a note of it like so:
```suggestion
*Following up on the previous question, what part of Neuro did you enjoy working on the most?*
```
or something similar